features/user_can_create_account.feature

```diff
@@ -0,0 +1,19 @@
+Feature: User can create account
+    As a user
+    In order to use the service
+    I need to be able to create an account
+
+    Background:
+        Given I am on the landing page
+        And I click on "Sign up"
+        Then I should be on the Sign up page
+
+    Scenario: Successfully creates an account
+        When I fill in "Name" with "Bob"
+        And I fill in "Email" with "bob@bob.bob"
+        And I fill in "Password" with "bobbobbob"
+        And I fill in "Password confirmation" with "bobbobbob"
+        And I click on "Create"
+        Then I should be on the landing page
+        And I should see "Welcome! You have signed up successfully."
+        And I should see "Hello, Bob"
```

Comment:
Please go ahead and test the multiple sad paths in this file as well.

Reply:
Yes. But will split it into more PRs.

Reply:
Nope nope, we will have the sad paths in this feature file, in this PR. 1 PT entry = 1 PR